packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/error.ts

```diff
@@ -45,15 +46,6 @@ export function processRegistryErrorList(
 
       operationOutcome.issue.push(issue);
     });
-
-    const msg = "RegistryErrorList is present in response";
-    capture.error(msg, {
-      extra: {
-        outboundRequest,
-        registryErrorList,
-        outboundRequestId: outboundRequest.id,
-      },
-    });
```

Comment:
removing archived capture which would be called multiple times per request if retries occurred